### ISSUE TITLE
siard --> oracle: loss of "foreign keys", "descriptions", "<nullable>true</nullable>

### ISSUE THREAD

**Issue by ibbenz** [sfa-siard/SiardGui#42]
After uploading a .siard file (V1.0), originally from an SQL-Database) into an Oracle-Database,
foreign keys aren't present anymore in the data. 
In addition all <descriptions> and <nullable>true</nullable>

The error can be reproduced using the following siar-File:
[3.1b_1-3_Northwind_simple_220913_B.zip](https://github.com/sfa-siard/SiardGui/files/9599457/3.1b_1-3_Northwind_simple_220913_B.zip)

During the upload following errors occured:
ORA-00904: "CUSTOMERID": invalid identifier; 
ORA-00904: "CATEGORYID": invalid identifier; 
ORA-00904: "ORDERID": invalid identifier

PS: I did the same transfer with the tool Fullconvert from Spectral Core.
There, the data (foreign keys, descriptions) 
were present in the Oracle-Database after upload.

**Comment by mburri** [comment]
Hey @ibbenz 
I didn't know about Fullconvert. Thanks for the tip - it seems to be a great tool. Does it also work with SIARD archives? 

How do you generate your SIARD 1.0 archives? We recently implemented support for SIARD 2.2 and while we try to convert 1.0 to 2.2. there was no special focus on this topic.

**Comment by ibbenz** [comment]
Yes Fullconvert works also with SIARD-archives (Database --> SIARD and SIARD --> Database). 

The SIARD 1.0 archives were generated by Claire Röthlisberger (KOST-CECO, Archivstrasse 24, Bern). I didn't ask her, which tool she used. The SIARD 1.0 archives were  successfully validated by the tool KOST-VAL and should be OK.

**Comment by ibbenz** [comment]
> Hey @ibbenz I didn't know about Fullconvert. Thanks for the tip - it seems to be a great tool. Does it also work with SIARD archives?
> 
> How do you generate your SIARD 1.0 archives? We recently implemented support for SIARD 2.2 and while we try to convert 1.0 to 2.2. there was no special focus on this topic.

The same issue occurs also in case of a Microsoft SQL Database
1. Uploading the siard 1.0 to Microsoft SQL --> no descriptions, but nullables are there

Nullables after upload by Siard Suite and Fullconvert:
![Nullable_Siard_Suite](https://user-images.githubusercontent.com/58251561/191770781-f27e9a24-6d33-459a-8a2a-d062a6e29f8f.png)
![Nullable_Fullconvert](https://user-images.githubusercontent.com/58251561/191770795-6f934bfe-cc14-4bc5-8812-1455953a6433.png)

Descriptions after upload by Siard Suite and Fullconvert:
![Description_Fullconvert](https://user-images.githubusercontent.com/58251561/191770788-1de625f6-2706-40ce-91c8-91b8b4340952.png)
![Description_Siard_Suite](https://user-images.githubusercontent.com/58251561/191770792-bf1068dc-6a19-47df-b856-cbed611ca210.png)


1. Downloading the data from Microsoft SQL Database --> the <nullable>true<nullable> disappears.

![Compare_Before_Upload_After_Download](https://user-images.githubusercontent.com/58251561/191771158-dfbbf0ca-0587-4d35-ba1f-3ea1bef14c3d.png)

**Comment by ibbenz** [comment]
**Bug also occurs with postgreSQL**
The Bug occurs also in case of the upload into a postgreSQL database. In addition to the loss of the "foreign keys" the "candidate keys" are also lost. 

I attached you the corresponding file. It differs slightly from the previously attached file, since I had to add an additional primary key for the column "OrderDetails" in order to be able to upload the file to postgreSQL.

**The testfile for postgreSQL:**
[0_1-3_Northwind_simple_220913_B_Suite3.zip](https://github.com/sfa-siard/SiardGui/files/9675482/0_1-3_Northwind_simple_220913_B_Suite3.zip)

**Comment by ibbenz** [comment]
This bug occurs as well with SIARD 1.0 files as with SIARD 2.1 files
